pipeline/tsimscripts_pipe.sh

```diff
@@ -85,6 +86,14 @@ do
                 shift
             done
             ;;
+        --dose)
```

Comment:
Here you make --dose to a required parameter. I would rather make it an optional paramter like defocus_lower and defocus_upper

Reply:
Addressed in latest commit.